appholder/src/main/java/com/android/identity/wallet/util/ProvisioningUtil.kt

```diff
@@ -205,10 +177,11 @@ class ProvisioningUtil private constructor(
         val mDocAuthOption = credential.applicationData.getString(MDOC_AUTHENTICATION)
         val settings = when (credential.credentialSecureArea) {
             is AndroidKeystoreSecureArea -> {
-                val keyInfo = credential.credentialSecureArea.getKeyInfo(credential.credentialKeyAlias) as AndroidKeystoreSecureArea.KeyInfo
+                val keyInfo =
+                    credential.credentialSecureArea.getKeyInfo(credential.credentialKeyAlias) as AndroidKeystoreSecureArea.KeyInfo
                 createAndroidKeystoreSettings(
                     keyInfo.isUserAuthenticationRequired,
-                    AddSelfSignedScreenState.MdocAuthStateOption.valueOf(mDocAuthOption),
+                    MdocAuthStateOption.valueOf(mDocAuthOption),
                     keyInfo.userAuthenticationTimeoutMillis,
                     keyInfo.userAuthenticationType,
                     keyInfo.isStrongBoxBacked,
```

Comment:
Looks like there's still some `AndroidKeystoreSecureArea` and `BouncyCastleSecureArea` specific code in here. That should all be moved to the relevant implementations of `SecureAreaSupport` interface.